src/store/RelayPendingQueryTracker.js

```diff
@@ -118,12 +118,14 @@ class PendingFetch {
   _errors: Array<?Error>;
 
   constructor(
+    /* eslint-disable comma-dangle */
```

Comment:
Don't strip flow types - babel-eslint parses them and we get the used/unused warnings. I think we just do this or leave it in there as a known failure (which makes lint a bit less useful though).

I would also update all the lint deps here to make sure you have latest eslint, babel-eslint (and then fix the version numbers - the ranges have been annoying in other projects as rules change).

Babel-eslint v5 (in beta) might fix this, but we should also file a test case against them so they can fix this.